tests/unit_tests/test_material.py

```diff
@@ -223,6 +223,39 @@ def test_isotropic():
     assert m2.isotropic == ['H1']
 
 
+def test_get_elements():
+    # test that zero elements exist on creation
+    m = openmc.Material()
+    assert len(m.get_elements()) == 0
+
+    # test addition of a single element
+    m.add_element('Li', 0.2)
+    assert len(m.get_elements()) == 1
+    assert 'Li' in m.get_elements()
+
+    # test that adding the same element
+    m.add_element('Li', 0.3)
+    assert len(m.get_elements()) == 1
+    assert 'Li' in m.get_elements()
+
+    # test adding another element
+    m.add_element('Si', 0.3)
+    assert len(m.get_elements()) == 2
+    assert 'Si' in m.get_elements()
```

Comment:
```suggestion
    assert m.get_elements() == ["Si", "Li"]
```

Reply:
Kind of, but it would actually remove the length check as well. And the sorted order would be ``["Li", "Si"]``

Reply:
good point